pkg/collector/resourceutilization/bpf/process_bpf_collector.go

```diff
@@ -89,6 +89,11 @@ func UpdateProcessBPFMetrics(bpfExporter bpf.Exporter, processStats map[uint64]*
 	for _, ct := range processesData {
 		comm := C.GoString((*C.char)(unsafe.Pointer(&ct.Comm)))
 
+		if ct.Pid == 0 && config.ExcludeSwapperProcess() {
```

Comment:
how about add a new metrics called idle cpu time?

Reply:
Yes, this is a good idea and aligns with  user / system / idle time in https://man7.org/linux/man-pages/man5/proc_stat.5.html

Reply:
@vimalk78 with this patch are you able to get sum(rate(kepler_process_bpf_cpu_time_ms_total[20s]))) == sum(rate(node_cpu_seconds_total{mode=~"user|system"}[20s]))

Reply:
with this change, under load condition
![image](https://github.com/user-attachments/assets/17ab6408-d8f9-4f93-9a87-206ded010c6a)